inc/add_site_ui/namespace.php

```diff
@@ -417,10 +417,10 @@ function handle_subdirectory( string $value ) : ?array {
 /**
  * Handle a custom domain input value.
  *
- * @param string $value Input custom domain value.
+ * @param string $url Input custom domain value.
  * @return array|null $value Assoc array with domain and path keys, or null if invalid.
  */
-function handle_custom_domain( string $value ) : ?array {
+function handle_custom_domain( string $url ) : ?array {
 	if ( strpos( $value, 'http' ) !== 0 ) {
 		$url = 'https://' . $value;
 	}
```

Comment:
The issue appears to be this bit of logic rather than the argument name - this change as it is will cause the same problem with `$value` being undefined. I think either this needs to change the `$value` variables to `$url` or have an `else` block that just contains `$url = $value;`

Reply:
Good point, sorry I missed that! I think if we just change the `$value` variables to `$url` we get the same outcome but without needing an `else` block that just sets the `$url` equal to `$value`. So, I did just that in my latest commit.